dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/ContentTypeResource.java

```diff
@@ -160,4 +161,74 @@ public final Response getRecentBaseTypes(@Context final HttpServletRequest reque
 
     return response;
   } // getTypes.
+
+  /**
+   * Return a list of {@link ContentType}, entity response syntax:.
+   *
+   * <code>
+   *  {
+   *      contentTypes: array of ContentType
+   *      total: total number of content types
+   *  }
+   * <code/>
+   *
+   * Url sintax: contenttype/query/query-string/limit/n-limit/offset/n-offset/orderby/fieldname-order_direction
```

Comment:
We should not use path params for filters. We should use query params for this instead: contenttype/?query=yourquery&limit=x&offnet=y&orderby=fieldname. Some ref http://stackoverflow.com/questions/11552248/when-to-use-queryparam-vs-pathparam